autogen/main/variables.tf.tmpl

```diff
@@ -889,3 +889,8 @@ variable "fleet_project_grant_service_agent" {
   default = false
 }
 {% endif %}
+variable "local_ssd_ephemeral_count" {
```

Comment:
Probably want to enclose in `{% if not autopilot_cluster %}` `{% endif %}`

Reply:
I changed variable name (because it was duplicated with `ephemeral_storage_config`), removed it from variables and added note to readme. I ran linter locally - seems to be fine.

Reply:
Thanks! Let's see how the tests come back.